src/main/java/pl/javastart/task/Room.java

```diff
@@ -0,0 +1,59 @@
+package pl.javastart.task;
+
+public class Room {
+    private double quaterMeters;
+    private double temperature;
+    private boolean airConditioner;
+    private final double minTemperature = 17.0;
+
+    public Room(double quaterMeters, double temperature, boolean airConditioner) {
+        this.quaterMeters = quaterMeters;
+        this.temperature = temperature;
+        this.airConditioner = airConditioner;
+    }
+
+    public boolean minusTemperature() {
+        if (airConditioner && temperature >= getMinTemperature() + 1) {
+            temperature--;
+        } else if (airConditioner && temperature > getMinTemperature()) {
+            setTemperature(17);
```

Comment:
Jeżeli masz dostęp do jakiegoś pola, to odwołuj się do niego bezpośrednio, a nie przez gettery i settery